starter-code/src/app/app.component.html

```diff
@@ -1,3 +1,11 @@
-<h1>
-  {{title}}
-</h1>
+<header id="site-header">
+
+</header>
+
+<div id="site-main" class="container">
+  <app-log-form-component></app-log-form-component>
```

Comment:
be careful and try to keep the structure of a router outlet pointing to pages and components contained inside those pages